scaleway/provider.go

```diff
@@ -192,6 +192,7 @@ func Provider() terraform.ResourceProvider {
 			"scaleway_instance_security_group_rules": resourceScalewayInstanceSecurityGroupRules(),
 			"scaleway_instance_server":               resourceScalewayInstanceServer(),
 			"scaleway_instance_placement_group":      resourceScalewayInstancePlacementGroup(),
+			"scaleway_iot_hub_beta":                  resourceScalewayIotHubBeta(),
```

Comment:
Should we avoid having beta in the name of the resource?

Reply:
I have done this to stick to the convention with k8s and lb, should the naming be related to the API or the state of the product?

Reply:
We add an internal debate about this as we foud out that migration from `beta` resource to non-beta is a HUGE pain for users. We would prefer and mandatory env variable to activate beta feature but it would not be coherent with other resources.

Reply:
I can do as you prefer ;)

Reply:
Changed it to remove the beta for now, if you want to keep it just ask ;)